### PR TITLE
fix: Fix missing DOM elements

### PR DESCRIPTION
**Pull request by dsimansk**

### CODE REVIEW

**Comment by dsimansk** [comment]
@jludvice I've probably fixed all outstanding issue, pls review it whenever you're available :)

File: e2e/integrations/integration.steps.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ class IntegrationSteps {
   @when(/^she selects "([^"]*)" integration action$/)
   public selectIntegrationAction(action: string): P<any> {
     const page = new ListActionsComponent();
+    if (action === 'Create Opportunity') {
+      log.warn(`Action ${action} is not available`);
+      return page.selectAction('Create Salesforce object');
```
Comment (jludvice) [inline e2e/integrations/integration.steps.ts]: what is this hack? :D

Comment (dsimansk) [reply e2e/integrations/integration.steps.ts]: It's kind of long to explain. There're a few occurrences of `Create Opportunity` action in scenarios, but it was removed from the selectable actions. I wanted to keep the scenarios and make the tests work with current state. But there was no value in that action anyway.

Comment (dsimansk) [reply e2e/integrations/integration.steps.ts]: @jludvice any more comments. I promise I'll fix it properly later :)

**Pull request merged**